src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/JsonFormat/NormalVerbosityJsonFormatSearchResultPackage.cs

```diff
@@ -6,17 +6,17 @@
 
 namespace NuGet.CommandLine.XPlat
 {
-    internal class NormalVerbosityPackage : MinimalVerbosityPackage
+    internal class NormalVerbosityJsonFormatSearchResultPackage : MinimalVerbosityJsonFormatSearchResultPackage
     {
-        [JsonPropertyName("total downloads")]
+        [JsonPropertyName("totalDownloads")]
```

Comment:
Rather than using a string, you should be able to use the JsonProperties from Protocol. 
I did this here: https://github.com/NuGet/NuGet.Client/blob/001ea1a/src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadata.cs#L67

Reply:
I also **strongly** encourage you to try to parse the json in multiple programming languages and get a feel for how to interact with the results.  Json property names with characters that are invalid variable names are very problematic in some languages, so if you want the json to be maximally useful to customers, you need to design the json schema to be maximally compatible with all these different restrictions.